flang/docs/FlangDriver.md

```diff
@@ -193,26 +190,29 @@ driver, `clang -cc1` and consists of the following classes:
 This list is not exhaustive and only covers the main classes that implement the
 driver. The main entry point for the frontend driver, `fc1_main`, is
 implemented in `flang/tools/flang-driver/driver.cpp`. It can be accessed by
-invoking the compiler driver, `flang-new`, with the `-fc1` flag.
+invoking the compiler driver, `flang`, with the `-fc1` flag.
 
 The frontend driver will only run one action at a time. If you specify multiple
 action flags, only the last one will be taken into account. The default action
 is `ParseSyntaxOnlyAction`, which corresponds to `-fsyntax-only`. In other
-words, `flang-new -fc1 <input-file>` is equivalent to `flang-new -fc1 -fsyntax-only
+words, `flang -fc1 <input-file>` is equivalent to `flang -fc1 -fsyntax-only
 <input-file>`.
 
 ## The `flang-to-external-fc` script
-The `flang-to-external-fc` wrapper script for `flang-new` was introduced as a
-development tool and to facilitate testing. The `flang-to-external-fc` wrapper
-script will:
-* use `flang-new` to unparse the input source file (i.e. it will run `flang-new
-  -fc1 -fdebug-unparse <input-file>`), and then
+The `flang-to-external-fc` wrapper script for `flang` was introduced as a
+development tool and to facilitate testing. While code-generation is not
+available in Flang, you can use it as a drop-in replacement for other Fortran
+compilers in your build scripts.
```

Comment:
The "While" sentence here is jarring to me. There _is_ code generation now?! Perhaps:
```suggestion
The `flang-to-external-fc` wrapper script for `flang` was introduced as a
development tool and to facilitate testing. In particular, it can be used to
delegate code generation to other Fortran compilers. Before Flang gained its
own code generation capabilities, this used to be the only way to create
actual binaries.
```

Reply:
Thank you for spotting this (yes, there _is_ code generation) and for the suggestion 🙏🏻  

I would actually like to reduce this paragraph and to eventually remove this script. So I will incorporate your suggestion only partially.